TACC/Core-CMS-Custom
# Task/apcd launch 2022 10 07 plus main since

## Overview

Merge APCD prod code (plus minor clean up) to `main`.

## Related

- many PRs to `epic/apcd-qa-2022-Q3`

## Changes

- add `submission_form` app

## Testing

<details><summary>Deployed & Tested on QA Server at 2022-10-19</summary>

Server: https://apcd-qa.tacc.utexas.edu/
Build: https://github.com/TACC/Core-CMS-Custom/actions/runs/3284573790
Deploy: https://jenkins01.tacc.utexas.edu/job/Core_Portal_Deploy/2080/

</details>

0. Log in to https://apcd-qa.tacc.utexas.edu/.
1. Load https://apcd-qa.tacc.utexas.edu/register/request-to-submit/. See:
    - **no** error
2. Submit https://apcd-qa.tacc.utexas.edu/register/request-to-submit/. See:
    - (if accurate data is submitted) **no** error
3. Load https://apcd-qa.tacc.utexas.edu/administration/list-registration-requests/. See:
    - manual content (i.e. not a custom app)

## UI

<details><summary>Table of Page Screenshots</summary>

| register > request submit<br><sub>filled out form</sub> | register > request submit<br><sub>form submit success</sub> | admin > list register.<br><sub>manual content test table</sub> |
| - | - | - |
| ![form - entries](https://user-images.githubusercontent.com/62723358/196792859-af182643-74ec-436c-b7de-6a46038a8f66.jpeg) | ![form - submit](https://user-images.githubusercontent.com/62723358/196792867-661f4cf3-0337-47c8-8a7c-91b1aa068e41.jpeg) | ![table - manual](https://user-images.githubusercontent.com/62723358/196792868-a1f6193c-4210-45a8-9189-0079d072e568.jpeg) |

</details>

## Notes

<details><summary>Process Used to Achieve this Diff</summary>

1. Checkout b959e6d _a.k.a. APCD Prod Launch 2022-10-07 @ approx. 18:00_.
2. Create branch `task/apcd-launch-2022-10-07-plus-main-since`.
3. Cherry Pick d16d69f _to remove print statements_.
5. Commit c3c8c10 _to **not** load custom app_.
6. Merge `main`.

</details>
